src/components/Set/Summary/index.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+
+import Accession from 'components/Accession';
+import Description from 'components/Description';
+import BaseLink from 'components/ExtLink/BaseLink';
+import { setDBs } from 'utils/processDescription/handlers';
+import Literature from 'components/Entry/Literature';
+
+import loadable from 'higherOrder/loadable';
+import config from 'config';
+import descriptionToPath from 'utils/processDescription/descriptionToPath';
+import ClanViewer from '../ClanViewer';
+
+import cssBinder from 'styles/cssBinder';
+
+import style from './style.css';
+import protvistaOptions from 'components/ProteinViewer/Options/style.css';
+import ipro from 'styles/interpro-vf.css';
+import summary from 'styles/summary.css';
+
+const css = cssBinder(style, summary, protvistaOptions, ipro);
+
+const SchemaOrgData = loadable({
+  loader: () => import(/* webpackChunkName: "schemaOrg" */ 'schema_org'),
+  loading: () => null,
+});
+
+export const schemaProcessData = ({
+  data: { accession, score },
+  db,
+}: {
+  data: { accession: string; score: number };
+  db: string;
+}) => ({
+  '@id': '@contains', // maybe 'is member of' http://semanticscience.org/resource/SIO_000095
+  name: 'entry',
+  value: {
+    '@type': ['Entry', 'StructuredValue', 'BioChemEntity'],
+    name: accession,
+    score,
+    url:
+      config.root.website.protocol +
+      config.root.website.href +
+      descriptionToPath({
+        main: { key: 'entry' },
+        entry: { db, accession },
+      }),
+  },
+});
+
+const SetLiterature = ({ literature }: { literature: Array<Reference> }) => {
+  if (!literature) return null;
+  const literatureEntries: [string, Reference][] = literature.map((ref) => {
+    const journalRegExp = /(.+) (\d{4});(\d+):(\d+-\d+)./;
+    const matches = journalRegExp.exec(ref.journal || '');
+    if (matches) {
+      return [
+        String(ref.PMID),
+        {
+          ...ref,
+          ISO_journal: matches[1],
+          year: Number(matches[2]),
+          volume: matches[3],
+          raw_pages: matches[4],
+        } as Reference,
+      ];
+    }
+    return [String(ref.PMID), ref];
+  });
+  return (
+    <>
+      <h4>Literature</h4>
```

Comment:
```suggestion
      <h4>References</h4>
```

Consistent with the entry summary page.